apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/Destination.java

```diff
@@ -143,7 +143,7 @@ public boolean hasContent() {
     @Override
     public void resetState() {
         address.setLength(0);
-        port = -1;
+        port = 0;
```

Comment:
why?

Reply:
It made a test fail. In general, when calling `resetState`, the instance should be in the same state as if it had just been instantiated. The field is initialized with the default value of `0`.